### PR TITLE
Enhance logging around snapshot create/delete

### PR DESCRIPTION
**Pull request by agrare**
Log the snapshot ref to INFO for both create and delete

https://bugzilla.redhat.com/show_bug.cgi?id=1549299

### CODE REVIEW

File: lib/VolumeManager/VolMgrPlatformSupportLinux.rb [hunk 1]
```diff
@@ -40,25 +40,27 @@ def preMount
...
     return unless @ost.force
     return unless @vimVm
 
     if @ost.force
       if !@snMor
-        $log.warn "VolMgrPlatformSupportLinux.postMount: VM not snapped: #{@cfgFile}"
+        $log.warn "#{log_prefix}: VM not snapped: #{@cfgFile}"
       else
-        $log.debug "VolMgrPlatformSupportLinux.postMount: removing snapshot snMor = \"#{@snMor}\""
+        $log.debug "#{log_prefix}: removing snapshot snMor = \"#{@snMor}\""
```
Comment (jerryk55) [inline lib/VolumeManager/VolMgrPlatformSupportLinux.rb]: Make this at least info to coincide with the call in preMount.

Comment (agrare) [reply lib/VolumeManager/VolMgrPlatformSupportLinux.rb]: I was thinking the pre/post symmetry was that they both logged to info the result after the operation:
`$log.info "VolMgrPlatformSupportLinux.preMount: VM snapshot [#{@snMor}] created in [#{Time.now - st}] seconds"`
and
`$log.info "#{log_prefix}: VM snapshot [#{@snMor}] removed"`
But I can just make this info.

Comment (agrare) [reply lib/VolumeManager/VolMgrPlatformSupportLinux.rb]: Or we can add a `INFO -- Creating snapshot` to preMount, thoughts?

Comment (jerryk55) [reply lib/VolumeManager/VolMgrPlatformSupportLinux.rb]: Oh you're right @agrare I didn't see the second line.

**Comment by agrare** [comment]
@roliveri just amended the commit to include the BZ link

**Comment by simaishi** [comment]
Fine backport (to manageiq-gems-pending repo) details:

```
$ git log -1
commit 32876723bd9de189e87cc7b5927aa38ac55843d7
Author: Richard Oliveri <oliveri.richard.github@gmail.com>
Date:   Tue Feb 27 13:07:00 2018 -0500

    Merge pull request #56 from agrare/additional_create_delete_snapshot_logging
    
    Enhance logging around snapshot create/delete
    (cherry picked from commit 24f925e9a7b8feb6c87fcd8542b7dd2e63ba1bb6)
```

**Comment by simaishi** [comment]
@miq-bot add_label fine/backported

**Pull request merged**